arch/arm/boot/dts/overlays/imx477_378-overlay.dtsi

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for IMX477 camera module on VC I2C bus
+/dts-v1/;
```

Comment:
This line and the next should only appear once - at the top of the including .dts files.